packages/contracts-sdk/src/implementations/ConsentFactoryContract.ts

```diff
@@ -46,6 +46,8 @@ export class ConsentFactoryContract
     return this.contractAddress;
   }
 
+  //#region Consent
```

Comment:
One of the regions is from the IConsentFactory.sol and the other is from IContentFactory.sol.

I found it really hard to keep track of what functions we've created sdks for and which we didn't. This will help us keep track in code much nicer. Plus they now come from different interfaces